app/views/admin/users/_user_friend_list.html.erb

```diff
@@ -0,0 +1,4 @@
+<h4>Has access to Friend records:</h4>
+<% @user.friend_list.each do |friend| %>
+  <%= link_to friend.name, friend_path(friend), target: "_blank" %>
```

Comment:
@adamesj this is my bad, the description was not specific about the 'friend record'.  The view I had in mind was the admin_edit_friend_path. I tested this out locally with a small change to visually separate the friend names:

<%= raw @user.friend_list.collect { |friend| link_to friend.name, edit_admin_friend_path(friend), target: "_blank" }.join(' | ') %>